src/views/figure_element.ts

```diff
@@ -142,10 +138,6 @@ export class FigureElementView extends BlockView<
 
         dispatch(tr)
       }
-
-      if (isModelFile(file)) {
-        await this.props.deleteModel(file.modelId)
-      }
```

Comment:
We need to delete the supplement node in this case.

Reply:
updated